SoundSwitch/UI/Forms/Settings.cs

```diff
@@ -690,14 +688,11 @@ private void RunAtStartup_CheckedChanged(object sender, EventArgs e)
         private void IconChangeChoicesComboBox_SelectedIndexChanged(object sender, EventArgs e)
         {
             if (!_loaded) return;
-            var comboBox = (ComboBox)sender;
+            var comboBox = (DisplayEnumObject<IconChangerEnum>)((ComboBox)sender).SelectedItem;
             if (comboBox == null) return;
 
-            var item = (DisplayEnumObject<IconChangerEnum>)iconChangeChoicesComboBox.SelectedItem;
-            AppConfigs.Configuration.SwitchIcon = item.Enum;
-            AppConfigs.Configuration.Save();
-
-            new IconChangerFactory().Get(item.Enum).ChangeIcon(AppModel.Instance.TrayIcon);
+            AppConfigs.Configuration.SwitchIcon = comboBox.Enum;
```

Comment:
```suggestion
            AppConfigs.Configuration.SwitchIcon = comboBox.Enum;
            AppConfigs.Configuration.Save();
```

Without this the change won't be saved since this isn't using the AppModel ^^'